### ISSUE TITLE
ETB-01 | Lack of Access Control for Creating Ethbridge Claims

### ISSUE THREAD

**Issue by Brando753** [Sifchain/sifnode#3079]
# Description
The function `GetCmdCreateEthBridgeClaim()` in `x/ethbridge/client/cli/tx.go` can be used to generate the command `create-claim` to create a claim on an Ethereum prophecy by the CLI command directly. The input parameters of this command are parsed in from the CLI by validators, which also lacks verifications from the actual claim information, see the finding MSR-01.

From the perspective of control flow, the function `CreateEthBridgeClaim()` should be only called by the module `cmd/ebrelayer`, instead of being called by the CLI command directly.

- First, the functionality of `GetCmdCreateEthBridgeClaim()` serves as a connection between Ethereum and SifChain by exchanging the prophecies. Calling the function solely by the module `cmd/ebrelayer` can reduce the input from validators manually, which further enhances the security.
- Second, if the validators can call the function from CLI, it is very likely to encounter various unexpected issues. For example, they have to track the nonce on their end to parse in the correct lock and burn sequence. In order guarantee to the correctness, certain validations are inevitable.

In conclusion, based on the control flow, it is safer and more logical to restrict the right of calling the function `CreateEthBridgeClaim()` to `cmd/ebrelayer` solely.

# Recommendation
The auditing team recommend removing the function `GetCmdCreateEthBridgeClaim()`.

**Comment by juniuszhou** [comment]
The function is already removed in future/peggy2, will create a pr in master branch to remove it.

**Comment by smartyalgo** [comment]
Added Jones, Caner, and Tanmay to the PR https://github.com/Sifchain/sifnode/pull/3110
